The AMI and OVA generation fails in v4.10.4-rc1

## Description

During the package generation for v4.10.4-rc1 in https://github.com/wazuh/wazuh-qa-automation/issues/7210, we have seen that the OVA and AMI builds have failed due to the following errors:

- AMI: https://github.com/wazuh/wazuh-virtual-machines/actions/runs/25905707025/job/76138850593

```
fatal: [***]: UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: Shared connection to *** closed.",
    "unreachable": true
}

PLAY RECAP *********************************************************************
*** : ok=28   changed=27   unreachable=1    failed=0    skipped=0    rescued=0    ignored=0   

Error: Process completed with exit code 4.
```

- OVA: https://github.com/wazuh/wazuh-virtual-machines/actions/runs/25905707556/job/76138850554

```
ERROR! [DEPRECATED]: community.general.yaml has been removed. The plugin has been superseded by the option `result_format=yaml` in callback plugin ansible.builtin.default from ansible-core 2.13 onwards. This feature was removed from community.general in version 12.0.0. Please update your playbooks.
ansible-playbook [core 2.16.0]
  config file = None
  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/runner/.local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/runner/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/runner/.local/bin/ansible-playbook
  python version = 3.10.12 (main, Mar  3 2026, 11:56:32) [GCC 11.4.0] (/usr/bin/python3)
  jinja version = 3.0.3
  libyaml = True
No config file found; using defaults
host_list declined parsing /tmp/allocatorvm_ova/inventory as it did not pass its verify_file() method
script declined parsing /tmp/allocatorvm_ova/inventory as it did not pass its verify_file() method
auto declined parsing /tmp/allocatorvm_ova/inventory as it did not pass its verify_file() method
Parsed /tmp/allocatorvm_ova/inventory inventory source with ini plugin
redirecting (type: callback) ansible.builtin.yaml to community.general.yaml
Error: ***[error]Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Updated the password tool with hash script modifications. ([#801](https://github.com/wazuh/wazuh-installation-assistant/pull/801))
 - The security of the installation assistant is improved when creating configuration files in LTS version. ([#723](https://github.com/wazuh/wazuh-installation-assistant/pull/723))
 
 ### Deleted

--- a/passwords_tool/passwordsFunctions.sh
+++ b/passwords_tool/passwordsFunctions.sh
@@ -205,7 +205,7 @@ function passwords_generateHash() {
         common_logger -d "Generating password hashes."
         for i in "${!passwords[@]}"
         do
-            nhash=$(bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/hash.sh -p "${passwords[i]}" 2>&1 | grep -A 2 'issues' | tail -n 1)
+            nhash=$(bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/hash.sh -p "${passwords[i]}" 2>&1)
             if [  "${PIPESTATUS[0]}" != 0  ]; then
                 common_logger -e "Hash generation failed."
                 if [[ $(type -t installCommon_rollBack) == "function" ]]; then
@@ -218,7 +218,7 @@ function passwords_generateHash() {
         common_logger -d "Password hashes generated."
     else
         common_logger "Generating password hash"
-        hash=$(bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/hash.sh -p "${password}" 2>&1 | grep -A 2 'issues' | tail -n 1)
+        hash=$(bash /usr/share/wazuh-indexer/plugins/opensearch-security/tools/hash.sh -p "${password}" 2>&1)
         if [  "${PIPESTATUS[0]}" != 0  ]; then
             common_logger -e "Hash generation failed."
             if [[ $(type -t installCommon_rollBack) == "function" ]]; then

